Simulator/loot_orb.py

```diff
@@ -0,0 +1,148 @@
+        'one_item': .6,
+        'champion_duplicator_one_gold_two_cost': .05
+    }
+    RARE = {
+        'ten_gold': .15,
+        # 'two_five_costs': .1,
+        'champion_duplicator_five_gold': .1,
+        'spatula': .75,
+    }
+
+# Implementation of the loot that can be given to the player
+# The rewards are pretty self explanatory; 'three_one_costs' gives three random one cost champions, 'one_item' gives one random item, etc.
+def give_loot(player, reward):
+    # I would use a match here but we're on python 3.8
+    if reward == 'three_gold':
+        player.gold += 3
+    if reward == 'six_gold':
+        player.gold += 6
+    if reward == 'ten_gold':
+        player.gold += 10
+    if reward == 'three_one_costs':
+        give_champions(player, pool_stats.COST_1, count=3)
+    if reward == 'two_cost_one_gold':
+        give_champions(player, pool_stats.COST_2)
+        player.gold += 1
+    if reward == 'three_cost':
+        give_champions(player, pool_stats.COST_3)
+    if reward == 'three_two_costs':
+        give_champions(player, pool_stats.COST_2, count=3)
+    if reward == 'two_three_costs':
+        give_champions(player, pool_stats.COST_3, count=2)
+    if reward == 'two_five_costs':
+        give_champions(player, pool_stats.COST_5, count=2)
+    if reward == 'one_item':
+        give_random_item(player)
+    if reward == 'full_item':
+        give_random_full_item(player)
+    if reward == 'champion_duplicator_one_gold':
+        player.gold += 1
+        player.add_to_item_bench('champion_duplicator')
+    if reward == 'champion_duplicator_one_gold_two_cost':
+        player.gold += 1
+        give_champions(player, pool_stats.COST_2)
+        player.add_to_item_bench('champion_duplicator')
+    if reward == 'champion_duplicator_five_gold':
+        player.gold += 5
+        player.add_to_item_bench('champion_duplicator')
+    if reward == 'spatula':
+        player.add_to_item_bench('spatula')
+
+
+# Utility Functions
+
+## Random Chapmions
+def give_champions(player, cost, count=1):
+    for _ in range(count):
+        give_champion(player, cost)
+
+def give_champion(player, cost):
+    # Give gold if bench is full
+    if player.bench_full():
+        if cost is pool_stats.COST_1:
+            player.gold += 1
+        if cost is pool_stats.COST_2:
+            player.gold += 2
+        if cost is pool_stats.COST_3:
+            player.gold += 3
+        if cost is pool_stats.COST_5:
+            player.gold += 5
+    else:
+        name = list(cost.items())[random.randint(0, len(cost) - 1)][0]
+        random_champion = champion.champion(name)
+        player.pool_obj.update_pool(random_champion, -1)
+        player.add_to_bench(random_champion)
+
+## Random Items
+def give_random_item(player):
+    item = player.random_item_from_pool()
+    player.add_to_item_bench(item)
+
+def give_random_full_item(player):
+    player.add_to_item_bench(item_list[random.randint(0, len(item_list) - 1)])
```

Comment:
Make sure this doesn't include any of the special items like kayn items or something.